src/api/__tests__/profile.api.spec.js

```diff
@@ -0,0 +1,57 @@
+import { GeoAdminProfile, ProfilePoint } from '@/api/profile.api'
+import { expect } from 'chai'
+import { describe, it } from 'vitest'
+
+const testProfile = new GeoAdminProfile([
+    new ProfilePoint(0, [0, 0], 100),
+    new ProfilePoint(50, [0, 50], 200),
+    new ProfilePoint(150, [0, 150], 90),
+    new ProfilePoint(200, [50, 150], 210),
```

Comment:
To better test the `maxElevation` on line 31, I would suggest to change the elevation order here to put the highest in the middle of the array.

Reply:
good point, I've added the last point so that I could compute a descent/ascent better but didn't think about the maxElevation, I'll switch it with second place.